test/util/ocm.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+## ocm.sh - Library to handle different ocm operations
```

Comment:
I renamed `servicelog.sh` to `ocm.sh` as well, as it gets the LS functionality. I also updated the api endpoint for the `ocm get /api/service_logs` as the old one was deprecated.

Reply:
I think renaming this shell script to `ocm.sh` is definitely appropriate here.